documentation/en/draft_modulators.md

```diff
@@ -0,0 +1,370 @@
+
+
+##### 2.5.2.2.1 Multiplexing of 2 different frequencies</h4>
+
+On the next figure we have two carriers with different frequencies to represent the different symbols inside the message, in this case 0 and 1. The frequency of the oscillator 1 must be different to the oscillator 2 but keeping in mind that the difference can't be too big.
+
+These two frequencies act like two phase digital switch controlled by a digital signal, in this case a digital message which generates an output shown in the switch.
+
+This technique has a drawback in which abrupt changes take place while switching the frequencies, such changes generate undesirable harmonics (sine wave) increasing the bandwidth, which is not desirable in signal modulation.
+
+<img src="imple_pic/modulator-switch.svg" alt="drawing" height="200" width="600" align="center"/>
+
+
+
+##### 2.5.2.2.2 Voltage Controlled Oscillator (VCO)
+
+<h3>VCO</h3>
+
+Instant frequency of the oscillator output its controlled by the input voltage. This kind of oscillator can produced a high frequency output on a wide range (few Hertz, or houndreds of Giga Hertz) depending on the DC input voltage asigned. 
+
+This type of oscillators doesn't presents any input voltage so it must oscillate in frequency called **suppressed oscillation frequency** _[needs reference]_ and by increasing the input voltage, the output signal will be altered in its frequency, linear growing regarding the input voltage.
+
+In this particular case in which we want to modulate a digital signal (pulse generation), we can observe two differents voltage to controll the VCO: when the data signal is on a high level or logic 1, its equivalent to having a voltage different from zero, and when the logic 0 is the current state is like having the VCO in the suppressed oscillation frequency for which it has been designed.
+
+This modulation technique, like the previous one, handles a different frequency for each symbol. The difference is the VCO deletes the abrupt changes in the frequency due it doesn't needs to multiplex between different frequency generators, it consists in controlling the the frequency coming from the generator.
+
+<img src="imple_pic/modulation-vco.svg" alt="drawing" height="250" width="800" align="center"/>
+
+
+#### 2.5.2.3 Gaussian Frequency Shift Keying (GFSK)
+
+<img src="imple_pic/Gaussian-filter.svg" alt="drawing" height="300" width="450" align="left"/>
+
+This technique is similar to the FSK, previously described, with an improvement on the signal bandwidth.
+
+The technique consists on soften the pulse generation (data or modulated signal), as we know such pulse generation represents a square wave which stands for abrupt changes when it passes from logic 0 to logic 1, increasing the bandwidth of the signal spectrum.
+
+Al pasar la señal moduladora (Data) a través de un filtro Gaussiano la salida es muy similar a una onda senoidal la cual presenta transiciones suaves de cero a uno.
```

Comment:
line 321 is the translation of the line 319, so we can delete 319

Reply:
no entiendo muy bien esto

Reply:
319 is in spanish and can be deleted, the the translated text is in the 321